cmake/modules/extensions.cmake

```diff
@@ -2498,8 +2498,8 @@ function(zephyr_get variable)
   cmake_parse_arguments(GET_VAR "MERGE" "SYSBUILD" "" ${ARGN})
 
   if(DEFINED GET_VAR_SYSBUILD)
-    if(NOT (${GET_VAR_SYSBUILD} STREQUAL "GLOBAL" OR
-            ${GET_VAR_SYSBUILD} STREQUAL "LOCAL")
+    if(NOT ("${GET_VAR_SYSBUILD}" STREQUAL "GLOBAL" OR
+            "${GET_VAR_SYSBUILD}" STREQUAL "LOCAL")
```

Comment:
what about in line 2524 and line 2539?

Reply:
Good catch, seems I forgot to force push before posting this, updated.